model/metrics/system-metrics.yaml

```diff
@@ -494,18 +494,18 @@ groups:
         examples: ["running"]
 
 
-  - id: metric.system.processes.count
+  - id: metric.system.process.count
     type: metric
-    metric_name: system.processes.count
+    metric_name: system.process.count
```

Comment:
Hum.. we may have a problem. I read again the metric guidelines we have today in https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#use-count-instead-of-pluralization-for-updowncounters and it says:

>If the value being recorded represents the count of concepts signified by the namespace then the metric should be named count (within its namespace). The pluralization rule does not apply in this case.

> For example if we have a namespace system.processes which contains all metrics related to the processes then to represent the count of the processes we can have a metric named system.processes.count. The suffix count here indicates that it is the count of system.processes.

So the `.count` one seems to be correct under `processes`. But I wonder about its attributes.. and also the `.created` one. Clearly one can't be in a namespace and the other in another. 🤔

Reply:
I believe this confusion was introduced by my #267. I sent #488 to fix it.

Reply:
So @trask is the renaming to `system.process.count` still valid? As https://github.com/open-telemetry/semantic-conventions/pull/488#discussion_r1381186138 indicates the example which still uses `system.processes` does not help to clarify the case 🤔

Reply:
I fixed #488, let me know if it's still unclear, thx!

Reply:
Thank's @trask that's clear now.
@joaopgrassi if you agree feel free to resolve this one.